packages/autocomplete/src/autocomplete.vue

```diff
@@ -216,8 +228,13 @@
       },
       close(e) {
         this.activated = false;
```

Comment:
_Info_ This handles the case where no selection is made, and a blur event is emitted. In this case, we wish to manually emit a blur to bypass the ignore next blur that was set.